init-letsencrypt.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v docker-compose)" ]; then
+  echo 'Error: docker-compose is not installed.' >&2
+  exit 1
+fi
+
+domains=(diamonds.etimo.se www.diamonds.etimo.se)
+rsa_key_size=4096
+data_path="./data/certbot"
+email="philip.forsberg@etimo.se"
+staging=1 # Set to 1 if you're testing your setup to avoid hitting request limits
+
+if [ -d "$data_path" ]; then
+  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
```

Comment:
Is it possible to use interactive shell functionality while running as a container? Wont this halt the container?

Reply:
This is just used for created a dummy cert so nginx can start. So we will just run this once outside the containers.

Reply:
Ok if it's run outside then it can be interactive